consensus/core/build.rs

```diff
@@ -84,6 +84,15 @@ fn build_tonic_services(out_dir: &Path) {
                 .codec_path(codec_path)
                 .build(),
         )
+        .method(
+            tonic_build::manual::Method::builder()
+                .name("get_latest_rounds_v2")
```

Comment:
We intentionally used protobuf / prost for request & response of GetLatestRounds. Can you try adding to that instead?

Reply:
Looks much better now, thanks! Go protobuf 😄